client/src/components/ElementEditor/HoverBar.js

```diff
@@ -63,9 +63,9 @@ class HoverBar extends Component {
   }
 
   toggle() {
-    this.setState({
-      popoverOpen: !this.state.popoverOpen
-    });
+    this.setState((prevState) => ({
```

Comment:
As above

Reply:
This is the rule [react/no-access-state-in-setstate](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-access-state-in-setstate.md).
```txt
Usage of this.state inside setState calls might result
in errors when two state calls are called in batch and thus referencing 
old state and not the current state.
``` 
This problem describes [there](https://legacy.reactjs.org/docs/react-component.html#setstate)
```txt
setState() does not always immediately update the component.
It may batch or defer the update until later. This makes reading this.state right after
calling setState() a potential pitfall. Instead, use componentDidUpdate or a setState callback
(setState(updater, callback)), either of which are guaranteed to fire after the update has been applied. 
If you need to set the state based on the previous state, read about the updater argument below.
```